.travis.yml

```diff
@@ -6,6 +6,14 @@
 language: php
 dist: trusty
 php: 7.0
+cache:
+  ccache: true
```

Comment:
It is a nice idea to use ccache to speed up the build but I do not see it being used anywhere in the build.

Reply:
hmmm, I thought it should be used as put its path the first in the `$PATH`: 
https://github.com/fossology/fossology/pull/1333/commits/13e2cfd39115de766fe212b7c400d475eb5087ba#diff-354f30a63fb0907d4ad57269548329e3R13

I'll print more detail to confirm that.

Reply:
You have added it to the `$PATH` but never used it anywhere.

You can do so by referring to the docs: https://ccache.dev/manual/latest.html#_run_modes and allow it to masquerade the GCC.

Reply:
The ccache package under Ubuntu automatically did that masquerade.

Reply:
(details can be found in https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/ccache/3.1.9-1/ccache_3.1.9-1.debian.tar.gz)